### PR TITLE
Archana_Assignement_34_Done_35mins

### PR DESCRIPTION
**Pull request by ArchanaNJun20**
Program 1 : 
Remove all Techno from given ArrayList.
image

Program 2 :
Display on which index Techno is available, Replace all Techno with Credits from given ArrayList, and count total number of Credits.
image

### CODE REVIEW

File: src/archanaNJun20/RemoveFromArrayList.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
+package archanaNJun20;
+
+import java.util.ArrayList;
+
+public class RemoveFromArrayList {
+	void removeTechno(ArrayList<String> input){
+		for(int index=0;index<input.size();index++){
```
Comment (AavrutiDJun20) [inline src/archanaNJun20/RemoveFromArrayList.java]: You can also directly write "for(;input.contains("Techno");)" like this

File: src/archanaNJun20/ReplaceTechnoWithCredits.java [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+				System.out.print(input.indexOf("Techno") + ",");
+				input.set(input.indexOf("Techno"), "Credits");
+			}
+		}
+		System.out.println();
+		return input;
+	}
+	void countWord(ArrayList<String> input){
+		int count=0;
+		for (int index = 0; index <= input.size(); index++){
```
Comment (AavrutiDJun20) [inline src/archanaNJun20/ReplaceTechnoWithCredits.java:23]: here also you can write like this "while(input.contains("Credits"))", as well as in above method "replaceTechnoWithCredits(ArrayList<String> input)"

**Review by AavrutiDJun20 — APPROVED** [review]
Good

**Review by KrishnaKTechnocredits — APPROVED** [review]
Good Job dear.

**Pull request merged**